bundles/org.palladiosimulator.metricspec.edit/META-INF/MANIFEST.MF

```diff
@@ -15,5 +16,6 @@ Require-Bundle: org.eclipse.core.runtime,
  de.uka.ipd.sdq.identifier;bundle-version="2.1.0";visibility:=reexport,
  de.uka.ipd.sdq.identifier.edit;bundle-version="2.1.0";visibility:=reexport,
  org.eclipse.emf.ecore;visibility:=reexport,
- org.eclipse.emf.ecore.edit;visibility:=reexport
+ org.eclipse.emf.ecore.edit;visibility:=reexport,
+ org.palladiosimulator.metricspecification.mwe2;bundle-version="5.0.0";resolution:=optional
```

Comment:
This does not consider any harm but seems unnecessary because the model bundle already has to have this dependency.